content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+Well, for most of the readers here this may not be the case anymore – we know, we are still young in our heart, but let’s be honest … 
+
+For sure, we all have those memories and definitely have certain associations by the word ‘Playground’.
+So what is it then with this thing called ‘mlrPlayground’?
+<br/>
+
+## The idea
+The thought behind this project was to offer a platform in the form of a <a href="https://shiny.rstudio.com/">Shiny</a> web application, where a user can try out different kinds of learners provided by the mlr package on a small set of destinct and tunable regression and classification tasks, making it possible to observe the prediction/performance behaviour based on changes on the task, the learner or the learner's hyperparameters.
+In the best case the user will gain new insights and deeper understanding of how each learner works, where it’s advantages are and also very importantly the cases when the learner fails miserably.
+There are a lot of different settings we want to offer in the user interface, and so – to not remove the fun of our playground – a huge effort went into making the UI as aesthetically pleasing and as smooth as possible.
+To achieve this, a website template was downloaded from <a href="https://templated.co/">Templated</a> and used as a baseline for design. After extending the template with missing CSS classes, most of the used shiny widgets have been overwritten – or even completely replaced –, offering refreshingly new visuals for the well known shiny framework.
+For the smoothness part, an object oriented <a href="https://cran.r-project.org/web/packages/R6/index.html">R6</a> class system with reactive attributes was engineered for the backend to give a well-defined framework of which elements should trigger what evaluation; an otherwise extremely tiresome and error-prone task for dozens of different UI elements.
+After all ‘mlrPlayground’ may not be as fun as a real playground, but you are also not as likely to hurt yourself and it is definitely more entertaining than looking at boring pictures of learners in a book.
+<br/>
+
+## The features
+
+In the following an extended overview of almost all available features is presented with the help of animated pictures.
+
+###### Task selection
+When running the app and scrolling down you can basically see two big boxes with content: the left one is for the task and learner settings, the right one for an overview of the learner’s bevahiour.
+For the first look it may be of help (even though not necessary) to explore the tasks.
+After finding the right button (I leave this task to the motivated reader) a panel with all available datasets is extended over our screen.
+Here, we can choose the tasktype of either classification with two covariables or regression with one covariable and several other parameters influencing the dataset: size, noise and the train/test split.
+
+<img src="https://i.imgur.com/ZV1wlBE.gif" width=800 />
```

Comment:
The GIF is great but the resolution could be better. Is it possible to create the GIF with a higher res?
While at it: It would be cool if there was no "windows watermark" in the bottom right corner.